components/TextFields/src/ContainedInputView/MDCBaseTextField.m

```diff
@@ -226,6 +226,19 @@ - (CGRect)clearButtonRectForBounds:(CGRect)bounds {
   return self.layout.clearButtonFrameNormal;
 }
 
+- (CGRect)placeholderRectForBounds:(CGRect)bounds {
+  if (self.shouldPlaceholderBeVisible) {
+    return [super placeholderRectForBounds:bounds];
+  }
+  return CGRectZero;
+}
+
+- (void)drawPlaceholderInRect:(CGRect)rect {
```

Comment:
please add a comment to the apple docs so that our future selves know that this is an override of an app API
https://developer.apple.com/documentation/uikit/uitextfield/1619637-drawplaceholderinrect?language=objc

Reply:
I can do that. Do we typically do that though? The fact that it calls `super` is usually enough to indicate that we're overriding something right?

Reply:
Do I need to add a link to Apple docs for every method that we override? Or just UITextField methods? Or just this one and `-placeholderRectForBounds:`?

Reply:
If we are not calling super on the first line it is helpful to give a hint to ourselves.

Or we can use a #pragma mark.
I see we have one for `UITextField Accessor Overrides`. I didn't see that. We can add one for non accessors.

Reply:
I just put `-drawPlaceholderInRect:` under a `UITextField Drawing Overrides` pragma mark 👍